src/jmh/java/net/sf/jabref/benchmarks/Benchmarks.java

```diff
@@ -52,6 +58,27 @@ public void init() throws IOException, SaveException {
                 new SavePreferences());
         bibtexString = stringWriter.toString();
 
+        List<String> latexSymbols = new ArrayList<>(HTMLUnicodeConversionMaps.UNICODE_LATEX_CONVERSION_MAP.values());
```

Comment:
I would prefer a simple hard-coded input. Mainly because it reduces code complexity. Moreover, it serves as a good baseline and then you might add some variations (like format of strings which need cleanup vs strings which don't).